website/src/pages/api/auth/[...nextauth].ts

```diff
@@ -222,9 +227,9 @@ export default function auth(req: NextApiRequest, res: NextApiResponse) {
              * in the frontend, when the user accepts the tos, we do a full refresh
              * which means this function will be called again.
              */
-            const { user_id, tos_acceptance_date } = await oasstApiClient.fetch_frontend_user(backendUser);
-            token.backendUserId = user_id;
-            token.tosAcceptanceDate = tos_acceptance_date;
+            const frontendUser = await oasstApiClient.upsert_frontend_user(backendUser);
```

Comment:
the backend does a lazy upsert so we should be safe. see https://github.com/LAION-AI/Open-Assistant/blob/6ba240f590787b40d56b3d14e19fd38b9488c850/backend/oasst_backend/api/v1/frontend_users.py#L62

Reply:
Oh sorry! I only saw this comment now.